merlin/utils.py

```diff
@@ -560,3 +586,19 @@ def convert_timestring(timestring: Union[str, int], format_method: str = "HMS")
     tdelta = convert_to_timedelta(timestring)
     LOG.debug(f"Timedelta object is: {tdelta}")
     return repr_timedelta(tdelta, method=format_method)
+
+
+def ws_time_to_td(ws_time: str) -> datetime:
```

Comment:
should it be 'dt' instead of 'td', as in short for 'datetime'?

Reply:
yes good catch